BlazarData/src/main/resources/schema.sql

```diff
@@ -1,197 +1,193 @@
 --liquibase formatted sql
```

Comment:
This just reverts back to mysql compliant syntax. H2 did not like using backticks, but not all column names are safe to use without quotes of some kind.